release/test_update_bundles_yml.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+source ../_liferay_common.sh
+source ../_test_common.sh
+source _publishing.sh
+
+function main {
+	setup
+
+	test_update_bundles_yml
+
+	tear_down
+}
+
+function setup {
+	export _PRODUCT_VERSION="2024.q3.1"
```

Comment:
Nice job, @lucasmiranda0!

Can you please also expand this test to cover the bi-weekly and GA releases? For example, 7.4.13-u130 and 7.4.3.125-ga125.

Thanks